docs/api-docs/channels/channels-toolkit-reference.md

```diff
@@ -34,44 +31,44 @@ All channels must integrate the following endpoints.
-| [Webhooks](https://developer.bigcommerce.com/api-reference/webhooks)                                                                   | Get notified when specific events occur in a BigCommerce store |
-| [Routes](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api)                                               | Create and manage page routes for headless storefronts         |
-| [Sites](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api)                                                | Create and manage sites associated with a channel              |
-| [Settings](https://developer.bigcommerce.com/api-reference/store-management/settings)                                                  | Read global and channel specific settings                      |
+| Endpoint                                                                                                    | Description                                                    |
+| ----------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------- |
+| [Channel Currency Assignments](https://developer.bigcommerce.com/api-reference/store-management/channels/channel-currency-assignments)|Manage channel specific currency settings|
+| [Price Lists](https://developer.bigcommerce.com/api-reference/store-management/price-lists)                 | Control variant-level pricing by channel, customer group, etc  |
+| [Store Information](https://developer.bigcommerce.com/api-reference/store-management/store-information-api) | Get store metadata                                             |
+| [Shipping](https://developer.bigcommerce.com/api-reference/store-management/shipping-api)                   | Manage how products are shipped                                |
+| [Webhooks](https://developer.bigcommerce.com/api-reference/webhooks)                                        | Get notified when specific events occur in a BigCommerce store |
+| [Routes](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api)                    | Create and manage page routes for headless storefronts         |
+| [Sites](https://developer.bigcommerce.com/api-reference/cart-checkout/sites-routes-api)                     | Create and manage sites associated with a channel              |
+| [Settings](https://developer.bigcommerce.com/api-reference/store-management/settings)                       | Read global and channel specific settings                      |
 
 ## Extended functionality endpoints
 
 The following endpoints provide extended functionality to channels.
 
-| Endpoint                                                                                                | Description                                                               |
```

Comment:
I will defer to @aglensmith to comment on how the tables are formatted in this article.